ChangeBGColor/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
```

Comment:
Bom nome! Podias ter sido especifico
```suggestion
    <title>ChangeBgColor</title>
```